golang/cosmos/x/swingset/client/cli/tx.go

```diff
@@ -44,11 +44,16 @@ func GetTxCmd(storeKey string) *cobra.Command {
 }
 
 // GetCmdDeliver is the CLI command for sending a DeliverInbound transaction
+// containing mailbox messages.
 func GetCmdDeliver() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "deliver [json string]",
-		Short: "deliver inbound messages",
-		Args:  cobra.ExactArgs(1),
+		Use:   "deliver {<messages JSON> | @- | @<file path>}",
```

Comment:
WDYT about using just "filename" instead of "file path"?

```suggestion
		Use:   "deliver {<messages JSON> | @- | @<filename>}",
```

Reply:
UNIX man pages seem to use just "file".